docs/docs/build/contracts/index.md

```diff
@@ -1,36 +1,74 @@
 # Contracts
 
-## BOB Testnet (Sepolia)
```

Comment:
Can you move the contract addresses to the end of the page, please? Also, keep this all under the `## BOB Testnet (Sepolia)` heading.

Thanks!